lib/openstudio-standards/standards/Standards.Motor.rb

```diff
@@ -0,0 +1,65 @@
+class Standard
+  # @!group Motor
+
+  # Determine the type of motor to model
+  #
+  # @param nominal_hp [Float] nominal or nameplate motor horsepower
+  # @return [String] motor type
+  def motor_type(nominal_hp)
+    return 'PSC'
+  end
+
+  # Determine the efficiency of fractional horsepower motors
+  #
+  # @param nominal_hp [Float] nominal or nameplate motor horsepower
+  # @param motor_type [String] motor type, either 'PSC' or 'ECM'
+  # @return [Float] motor efficiency
```

Comment:
Return type in the documentation shall be `[Hash, NilClass]`

Reply:
That's right, I'll change it.